geowebcache/wms/src/main/java/org/geowebcache/service/wms/WMSService.java

```diff
@@ -287,20 +291,22 @@ public void handleRequest(Conveyor conv) throws GeoWebCacheException {
                 WMSGetCapabilities wmsCap = new WMSGetCapabilities(tld, tile.servletReq, servletBase, context, urlMangler);
```

Comment:
No security for getCapabilities?

Reply:
That's certainly something we should do in the future but at the moment I'm just trying to get the existing behaviour done in a better way.